.gitignore

```diff
@@ -1,3 +1,6 @@
 *.swp
 bin/
 vendor/
+*.DS_Store
+*.idea
+test/e2e/results/
```

Comment:
Could you prepend this with `/` so that it's clear only the root folder is filtered not ./abc/test/e2e/results?